jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -406,6 +406,14 @@ void java_bytecode_convert_method_lazy(
   // Not used in jbmc at present, but other codebases that use jbmc as a library
   // use this information.
   method_symbol.type.set(ID_C_abstract, m.is_abstract);
+
+
+  if(java_bytecode_parse_treet::find_annotation(
+       m.annotations, "java::org.cprover.MustNotThrow"))
```

Comment:
5-space indent?

Reply:
2 w.r.t. the start of the `java_bytecode_parse_treet::find_annotation(` statement I guess